### PR TITLE
add disabled section to button docs

### PR DESCRIPTION
**Pull request by wibjorn**
Link: preview-372

https://design.docs.microsoft.com/pulls/372/components/button.html

## Testing

Read button docs

### CODE REVIEW

File: site/src/components/button.md [hunk 1]
```diff
@@ -23,11 +23,11 @@ All buttons, by default, are secondary buttons. There are three graduating secon
 2. Clear
 3. Filled
 
-| Type     | Class                      | Default State                                         | Hover                                                            |
-| -------- | -------------------------- | ----------------------------------------------------- | ---------------------------------------------------------------- |
-| Outlined | `.button`                  | <button class="button">Default</button>               | <button class="button is-hovered">Default</button>               |
-| Clear    | `.button` `.button-clear`  | <button class="button button-clear">Default</button>  | <button class="button button-clear is-hovered">Default</button>  |
-| Filled   | `.button` `.button-filled` | <button class="button button-filled">Default</button> | <button class="button button-filled is-hovered">Default</button> |
+| Type     | Class                      | Default State                                        | Hover                                                           |                                                                          |
```
Comment (homing1) [inline site/src/components/button.md:26]: Missing a `Disabled` header

File: site/src/components/button.md [hunk 2]
```diff
@@ -92,11 +92,11 @@ Warning buttons are yellow on most themes. This color indicates a critical situa
...
+| Type     | Class                                           | Default State                                                       | Hover                                                                          | Disabled                                                                     |
+| -------- | ----------------------------------------------- | ------------------------------------------------------------------- | ------------------------------------------------------------------------------ | ---------------------------------------------------------------------------- |
+| Outlined | `.button`, `.button-warning`                    | <button class="button button-warning">Default</button>              | <button class="button button-warning is-hovered">Default</button>              | <button class="button button-warning" disabled>Default</button>              |
+| Clear    | `.button`, `.button-warning`, `.button-clear`   | <button class="button button-warning button-clear">Clear</button>   | <button class="button button-warning button-clear is-hovered">Clear</button>   | <button class="button button-warning button-clear" disabled>Default</button> |
```
Comment (homing1) [inline site/src/components/button.md:98]: Disabled Clear button should be "Clear" instead of "Default"

Comment (wibjorn) [reply site/src/components/button.md:98]: 👍 Updated

File: site/src/components/button.md (hunk 1 above)
Comment (wibjorn) [reply site/src/components/button.md:26]: 👍 Updated

**Review by ollips — APPROVED** [review]

**Review by homing1 — APPROVED** [review]

**Pull request merged**